auth_totp/README.rst

```diff
@@ -0,0 +1,103 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========
+MFA Support
+===========
+
+This module adds support for MFA using TOTP (time-based, one-time passwords). 
+It allows users to enable/disable MFA and manage authentication apps/devices 
+via the "Change My Preferences" view and an associated wizard. After logging 
+in normally, users with MFA enabled are taken to a second screen where they 
+have to enter a password generated by one of their authentication apps and 
+are presented with the option to remember the current device. This creates a 
+secure, HTTP-only cookie that allows subsequent logins to bypass the MFA step.
+
+Installation
+============
+
+To install this module, simply follow the standard install process, making 
+sure that the PyOTP library is available in your environment. The module will 
+not function properly without PyOTP, which can easily be added via pip: 
+``pip install pyotp``.
+
+Configuration
+=============
+
+By default, the trusted device cookies introduced by this module have a 
+``Secure`` flag and can only be sent via HTTPS. You can disable this by going 
+to ``Settings > Parameters > System Parameters`` and changing the 
+``auth_totp.secure_cookie`` key to ``0``, but this is not recommended in 
+production as it increases the likelihood of cookie theft via eavesdropping.
+
+Usage
+=====
+
+Install and enjoy.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/9.0
+
+Known Issues / Roadmap
+======================
+
+Known Issues
+------------
+
+* Due to an Odoo bug, the module does not uninstall cleanly, leaving the 
```

Comment:
I believe that comma is needed there (http://www.wheaton.edu/Academics/Services/Writing-Center/Writing-Resources/The-Comma) given how the sentence is currently constructed, but I can get rid of it by writing the following instead: `The module does not uninstall cleanly due to an Odoo bug`